package.json

```diff
@@ -142,6 +144,10 @@
     "arrowParens": "avoid",
     "trailingComma": "none"
   },
+  "engines": {
+    "node": ">= 14",
```

Comment:
Will this make npm yell when people are installing SWR to their projects, when they have node < 14?

Reply:
Yes it will. Do you have any concern about this?

Reply:
Emmm yes, probably we don't need node sincei it's not using any specific node api